javascript-challenges.test.js

```diff
@@ -39,5 +63,54 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+describe("fullerHouse", () => {
+    it("function that takes in an array of 5 numbers and determines whether or not the array is a 'full house'", () => {
+        expect(fullerHouse(hand1)).toEqual(true);
+        expect(fullerHouse(hand2)).toEqual(false);
+        expect(fullerHouse(hand4)).toEqual(true);
+        expect(fullerHouse(hand3)).toEqual(false);
+    });
+  });
+
+
+
 
 // Pseudo code:
+
+
+// Create a function called fullerHouse.
+// Use math.random to give numbers between 2-7.
+// Create a for loop to determine if 3 of the numbers are the same and return true it is and false if its not. 
```

Comment:
Your test is handing the function an argument of a particular hand in the form of an array.  This should be what you are evaluating against instead of a randomized number.  Please make sure to reach out if you are stuck or need clarification on the problem.  This is an important step in development.